pkg/webhook/acceleratednetwork/add.go

```diff
@@ -27,5 +28,10 @@ func AddToManager(mgr manager.Manager) (*extensionswebhook.Webhook, error) {
 			{Obj: &appsv1.DaemonSet{}},
 		},
 		Mutator: NewMutator(mgr, logger),
+		ObjectSelector: &metav1.LabelSelector{
+			MatchLabels: map[string]string{
+				"k8s-app": "calico-node",
```

Comment:
Why not use this constant for the key?
https://github.com/gardener/gardener/blob/98c33a73be257e03f48da29a8dab5613905d15a0/pkg/component/networking/coredns/constants/constants.go#L9